### PR TITLE
Bug 1832322 add module for changing registry ManagementState to install guide

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1832322

### CODE REVIEW

**Comment by bmcelvee** [comment]
Preview: https://bz1832322--ocpdocs.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html#installation-registry-storage-config_installing-restricted-networks-bare-metal

**Comment by bmcelvee** [comment]
@openshift/team-documentation please peer review, thanks!

File: installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc [hunk 1]
```diff
@@ -92,6 +92,8 @@ include::modules/installation-operators-config.adoc[leveloffset=+1]
 
 include::modules/installation-registry-storage-config.adoc[leveloffset=+2]
 
+include::modules/registry-change-management-state.adoc[leveloffset=+3]
```
Comment (bergerhoffer) [inline installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc:95]: @bmcelvee I know you're only including the file that already exists, but I wasn't sure if it was already peer reviewed or not. So just a few quick thoughts on it in case you care to take a look:

* For the title of the module, suggest Change -> Changing
* The way it's worded seems a little confusing. Is the operator called the "ManagementState Image Registry Operator"? Or are you just changing the managementState field of the Image Registry Operator? If the latter, I'd suggest rewording throughout the module to something a little clearer. For example, "Changing the image registry's management state" for the title, and "To start the image registry, you must change the Image Registry Operator configuration's `managementState` from `Removed` to `Managed`."

**Review by bergerhoffer — COMMENTED** [review]
The module addition LGTM, just a few comments on the actual contents of the module that already existed.

File: installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc (hunk 1 above)
Comment (bmcelvee) [reply installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc:95]: Thank you! It was peer reviewed a while back, but I don't mind updating it.
